1.Exploratory_Data_Analysis/scripts/2.extended_umap_analysis.ipynb

```diff
@@ -0,0 +1,612 @@
+{
```

Comment:
<p>Maybe this is mentioned somewhere else, but consider commenting on the purpose of the analysis and and the value of the models in this case</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/WayScience/pyroptosis_signature_data_analysis/pull/45/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>